go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -31,6 +31,7 @@ import (
 	"vitess.io/vitess/go/constants/sidecar"
 	"vitess.io/vitess/go/mysql/replication"
 	"vitess.io/vitess/go/mysql/sqlerror"
+	"vitess.io/vitess/go/vt/schema"
 	"vitess.io/vitess/go/vt/vtenv"
 
```

Comment:
Can we delete this spacer line so all "vitess.io/vitess/" sort together?

Reply:
Done! Good catch. I do try to clean up the imports whenever I edit a file but missed this one.